api/taxonomies/serializers.py

```diff
@@ -19,20 +19,29 @@ class TaxonomySerializer(JSONAPISerializer):
     filterable_fields = frozenset([
         'text',
         'parents',
+        'parent',
         'id'
     ])
     id = ser.CharField(source='_id', required=True)
     text = ser.CharField(max_length=200)
-    parents = JSONAPIListField(child=TaxonomyField())
+    parents = ser.SerializerMethodField()
```

Comment:
`parents` should be deprecated. Maybe just add a comment for now, unless @brianjgeiger has a different way to mark fields as deprecated.